include/psa/crypto_extra.h

```diff
@@ -121,6 +121,15 @@ psa_status_t mbedtls_psa_inject_entropy(const unsigned char *seed,
                                         size_t seed_size);
 
 
+/* The Number of key slots (plus one because 0 is not used).
+ * The value is a compile-time constant for now, for simplicity.
+ * Note: this configuration flag is an implementation detail that may change
+ * or be removed in future versions */
+#if !defined(MBEDTLS_PSA_KEY_SLOT_COUNT)
+    #define MBEDTLS_PSA_KEY_SLOT_COUNT 32
```

Comment:
how about default number of 16 or 20?
cc @gilles-peskine-arm

Reply:
as was mentioned in the email discussion, we need to see some testing on real world use (e.g. mbed client usage) to decide on a new default value. So for now I've left the default as it is. If we can decide on a new value I will set it.